jobs/definition/constantly.sh

```diff
@@ -0,0 +1,2 @@
+git pull
```

Comment:
I would put `#!/bin/sh` in all of the shell scripts. It's not strictly necessary, but it's best practice.